_posts/2024-10-24-acoustics-neurips-2024.md

```diff
@@ -0,0 +1,13 @@
+---
+date: 2024-10-11 00:00:00-07:00
+description: null
+featuredImage: assets/ASID_NeurIPS_logo.png
+author: Sean
+layout: blog-post
+slug: /acoustics-neurips-2024
+title: Acoustics Species Will be at NeurIPS 2024!
+categories:
+- acoustic-species-id
+---
+
+Our paper, "A Deep Learning Approach to the Automated Segmentation of Bird Vocalizations from Weakly Labeled Crowd-sourced Audio" has been accepted for poster presentation for Cliamte AI's Tackling Climate Change workshop in NeurIPS 2024! Congratulations to the authors, Jacob Ayers, Sean Perry, Samantha Prestrelski, Tianqi Zhang, Ludwig von Schoenfeldt, Mugen Blue, Gabriel Steinberg, Mathias Tobler, Ian Ingram, Curt Schurgers, and Ryan Kastner!
```

Comment:
Recommend changing to `Congratulations to the authors: Jacob Ayers, Sean Perry...`